packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -13,8 +13,16 @@ import 'forces.dart';
 import 'listener_helpers.dart';
 import 'ticker.dart';
 
+/// A controller for an animation.
+///
+/// A animation controller can drive an animation forward or backward and can
+/// set the animation to a particular value. The controller also defines the
+/// bounds of the animation and can drive an animation using a physics
+/// simulation.
 class AnimationController extends Animation<double>
-  with EagerListenerMixin, LocalAnimationListenersMixin, LocalAnimationStatusListenersMixin {
+  with EagerListenerMixin, LocalListenersMixin, LocalAnimationStatusListenersMixin {
+
+  /// Creates an animation controller.
```

Comment:
Maybe include details about what 'value' is, why 'debugLabel' might be useful (it's included in various toStrings), how to use lowerBound and upperBound, whether those have to be ordered (is upper<lower ok?).